server/src/threads.cpp

```diff
@@ -153,8 +153,8 @@ static void *th_noise_hdl(void *args) {
 
     while(1) {
         read(pctx->noise_r_fd, &show_ready, sizeof show_ready);
-        cv::namedWindow("Noise Visualizer", cv::WINDOW_GUI_NORMAL);
-        cv::imshow("Noise Visualizer", *(show_ready->nframe));
+        cv::namedWindow("Black And White Visualizer", cv::WINDOW_GUI_NORMAL);
```

Comment:
This is not only used for black and white visualization; better use a more generic name

Reply:
Yes, you are right. I changed it during testing, but it's better to change it

Reply:
Changed it to "Modified Image Visualizer"